### PR TITLE
Update track descriptions for beta-3

### PR DESCRIPTION
**Pull request by veidenberg**
### Description
Updates for beta-3:
* Update species-specific track descriptions
* Update track templates
* Improve track submission script (e.g. add logfile)
* Include the script for generating gene track CSV file ([source repo](https://github.com/Ensembl/ensembl-common/blob/sgiorgetti/master/scripts/new_gb_beta/get_analysis_descs_beta2.py))
* Remove unused templates

### Related JIRA Issue(s)
[ENSWBSITES-2798](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2798)

### CODE REVIEW

File: templates/variant-track-desc.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Genome_UUID,Track_name,Source_name,Source_URL,Description
+a733574a-93e7-11ec-a39d-005056b38ce3,SGRP short variants,SGRP,http://www.moseslab.csb.utoronto.ca/sgrp,"Variation features from SGRP, with Ensembl identifiers"
```
Comment (nakib103) [inline templates/variant-track-desc.csv:2]:
```suggestion
a733574a-93e7-11ec-a39d-005056b38ce3,EVA short variants,EVA,https://www.ebi.ac.uk/eva,"All short variants (SNPs and indels) data from European Variation Archive (EVA) - release 6"
```

Or probably we do not need this here as it is standard EVA source.

File: templates/variant-track-desc.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+08c83543-c3d9-4a7a-85d8-52f75d61aab0,CSHL/Cornell short variants,CSHL/Cornell,,"All short variants (SNPs and indels) data from CSHL/Cornell"
+bc9cb479-b1e3-460b-b4ac-d20b0fe1b350,CNR-ITB short variants,CNR-ITB,,"All short variants (SNPs and indels) data from CNR-ITB"
+35a40586-9cb2-4d89-a7db-d00511691d09,The 150 TGRSP short variants,150 Tomato Genome ReSequencing Project,https://www.tomatogenome.wur.nl,All short variants (SNPs and indels) data from the 150 Tomato Genome ReSequencing Project
+a73357ab-93e7-11ec-a39d-005056b38ce3,CSHL/Cornell short variants,"CerealsDB, EMS-induced mutation, Inter-homoeologous, Nottingham_WRC, Watkins-exome-capture, Exome_Capture_Diversity",,All short variants (SNPs and indels)
```
Comment (nakib103) [inline templates/variant-track-desc.csv:6]:
```suggestion
a73357ab-93e7-11ec-a39d-005056b38ce3,Short variants (all sources),"CerealsDB, EMS-induced mutation, Inter-homoeologous, Nottingham_WRC, Watkins-exome-capture, Exome_Capture_Diversity",,All short variants (SNPs and indels)
```

**Review by nakib103 — CHANGES_REQUESTED** [review]
Hi @veidenberg , 
I requested some changes that seems to have mismatched data for tracks.
ref: `/nfs/production/flicek/ensembl/variation/new_website/v110/payloads/non_human_track_metadata.json`

File: templates/variant-track-desc.csv (hunk 1 above)
Comment (veidenberg) [reply templates/variant-track-desc.csv:2]: Good spot. Yes, we can just remove this CSV row as it doesn't differ from the eva track template.

File: templates/variant-track-desc.csv (hunk 2 above)
Comment (veidenberg) [reply templates/variant-track-desc.csv:6]: CSV updated.

Comment (veidenberg) [reply templates/variant-track-desc.csv]: CSV updated.

**Review by nakib103 — APPROVED** [review]

**Comment by nakib103** [comment]
@veidenberg, thanks the csv file looks alright now.

**Pull request merged**